quic/s2n-quic-tls-default/Cargo.toml

```diff
@@ -15,3 +15,10 @@ s2n-quic-tls = { version = "=0.30.0", path = "../s2n-quic-tls" }
 
 [target.'cfg(not(unix))'.dependencies]
 s2n-quic-rustls = { version = "=0.30.0", path = "../s2n-quic-rustls" }
+
+[target.'cfg(unix)'.dev-dependencies]
+jobserver = "=0.1.26" # newer versions require rust 1.66, see https://github.com/aws/s2n-quic/issues/1991
+
+# jobserver is just here to pin
```

Comment:
I don't know that I would understand this comment if we hadn't already talked about it. 
```suggestion
# jobserver is pinned here, because the cargo package process doesn't respect pins on transitive dev depenedencies
```

Reply:
agree, though the comment on line 22 is referring to why we need udeps to ignore this dependency (it detects it as unused). I'll update the comment on line 20 and the corresponding pin in s2n-quic/Cargo.toml